e2e/keywords/sharemanager.resource

```diff
@@ -35,3 +35,23 @@ Wait for sharemanager pod of deployment ${deployment_id} running
     ${deployment_name} =   generate_name_with_suffix    deployment    ${deployment_id}
     ${volume_name} =    get_workload_volume_name    ${deployment_name}
     wait_for_share_manager_pod_running    ${volume_name}
+
+Assert disk size in sharemanager pod for ${workload_kind} ${workload_id} is ${size}
```

Comment:
The following assertion is target to encrypted shared volume (see the reason below)

```suggestion
Assert encrypted disk size in sharemanager pod for ${workload_kind} ${workload_id} is ${size}
```